scripts/manpage_generation.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Directory to store the generated manpages
+MANPAGE_DIR="man"
+
+VER=$(grep -oP '^Version:\s+\K\S+' packaging/convert2rhel.spec)
+
+# Generate the manpage using argparse-manpage
+PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "$VER" --prog="convert2rhel" --include man/distribution --include man/synopsis > "$MANPAGE_DIR/convert2rhel.8"
```

Comment:
```suggestion
# Generate a file with convert2rhel synopsis for argparse-manpage
python -c 'from convert2rhel import toolopts; print("[synopsis]\n."+toolopts.CLI.usage())' > man/synopsis

# Generate the manpage using argparse-manpage
PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "convert2rhel $VER" --prog="convert2rhel" --include man/distribution --include man/synopsis > "$MANPAGE_DIR/convert2rhel.8"
```
The project name has been till now `convert2rhel <ver>`, not just `<ver>`.

Reply:
I have just made this convert2rhel_ver and not what was suggested I will fix that and push!

Reply:
done!